### PR TITLE
Globbing.py: Fix docstring typo

### PR DESCRIPTION
**Pull request by anubhawbhalotia**
This fixes the typo and changes it from obtaines --> obtained.

Closes https://github.com/coala/coala/issues/5920

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by anubhawbhalotia** [comment]
> @anubhawbhalotia, working on multiple newcomer's issues is advised against. You should probably take up some difficulty/low issues next.

Yea, sure

File: coalib/parsing/Globbing.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def _iter_choices(pattern):
 @yield_once
 def _iter_alternatives(pattern):
     """
-    Iterates through all glob patterns that can be obtaines by combination of
+    Iterates through all glob patterns that can be obtained by combination of
```
Comment (utkarsh2102) [inline coalib/parsing/Globbing.py]: Though the PR is ready as per the issue, but could you possible amend the missing articles, too?

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5956/commits/262f85283d0ac6a8ccfc609f8662bf084978d4a3

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**